WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1478,6 +1478,23 @@ ContactListPaginationChild          = display:inline-block;
 ContactListPaginationLink           = 
 ContactListPaginationLinkIcon       = height:30px; width:30px;
 
+# Contact Search
+
+ContactSearchListContainerHeight       = 300px;
+ContactSearchResultsContainer          = margin-top:@SkinWrapperPadding@; @PanelBorder@
+ContactSearchListColHeader             = display:none;
+ContactSearchListContainer             = @FullWidth@; @PageBg@; @ContactSearchListContainerHeight@;
+ContactSearchInputField                = @InputPadding@; @ButtonHeight@; border-width: 0 1px 0 0; width:93%;
+ContactSearchListRow                   = @GroupEditContentListRow@;
+ContactSearchListRowSelected           = @GroupEditContentListRowSelected@;
+ContactSearchContactGroupItemContent   = @ContactGroupItemContent@
+ContactSearchListRowItems              = @GroupEditContentListRowText@
+ContactSearchDialogOuterContainer      = @DialogBackGroundColor@
+ContactSearchQueryWidget               = @FullWidth@; @PanelBorder@;
+ContactSearchButton                    = float:right; width:7%; @ButtonHeight@;
```

Comment:
the logic of having 7 % here is that the other 93 % is consumed  by Search Input field.
So, going with %.
We have also taken the width in % attach contacts model, https://github.com/Zimbra/zm-zimlets/blob/feature/universal-ui/src/zimlet/com_zimbra_attachcontacts/attachcontacts.css#L63